server/server.js

```diff
@@ -11,3 +11,32 @@
 // iterating through that list, display the string you built up.
 
 // Remember that the response is a stream object that must be closed.
+
+var http = require('http');
+var enhancedDate = ('./enhancedDate');
```

Comment:
missing a require() function call here - enhancedDate is just being assigned a string here